app/controllers/clearance/users_controller.rb

```diff
@@ -11,8 +12,13 @@ class Clearance::UsersController < Clearance::BaseController
 
   def index
     if signed_in?
+      if params[:search]
```

Comment:
I think what hound is trying to say here is that its better to be explicit and do something like `if params[:search].present?`